Sources/Core/Screen/ScreenCapturedMonitor.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  ScreenCapturedMonitor.swift
+//  XestiMonitors
+//
+//  Created by Paul Nyondo on 2018-04-06.
+//
+//  © 2018 J. G. Pusey (see LICENSE.md).
+//
+
+#if os(iOS) || os(tvOS)
+
+    import UIKit
```

Comment:
Please move all code over to left margin -- we are no longer indenting it between `#if`/`#endif` pairs.

Reply:
I have changed that.